ompi/mpi/man/man3/MPI_Comm_compare.md

```diff
@@ -0,0 +1,65 @@
+# Name
+
+`MPI_Comm_compare` - Compares two communicators.
+
+# Syntax
+
+## C Syntax
+
+```c
+#include <mpi.h>
+
+int MPI_Comm_compare(MPI_Comm comm1, MPI_Comm comm2, int *result)
+```
+
+## Fortran Syntax
+
+```fortran
+USE MPI
+! or the older form: INCLUDE 'mpif.h'
+
+MPI_COMM_COMPARE(COMM1, COMM2, RESULT, IERROR)
+    INTEGER	COMM1, COMM2, RESULT, IERROR
+```
+
+## Fortran 2008 Syntax
+
+```fortran
+USE mpi_f08
+
+MPI_Comm_compare(comm1, comm2, result, ierror)
+    TYPE(MPI_Comm), INTENT(IN) :: comm1, comm2
+    INTEGER, INTENT(OUT) :: result
+    INTEGER, OPTIONAL, INTENT(OUT) :: ierror
+```
+
+
+# Input Parameters
+
+* `comm1` : Comm1 (handle).
+* `comm2` : Comm2 (handle).
+
+# Output Parameters
+
+* `result` : Result of comparison (integer).
+* `IERROR` : Fortran only: Error status (integer).
+
+# Description
+
+`MPI_IDENT` `result`s if and only if `comm1` and `comm2` are handles for the
+same object (identical groups and same contexts). `MPI_CONGRUENT` `result`s
```

Comment:
Looks like `result` got caught up in the regexp for quoting, even though "results" in this case isn't really a use of the parameter `result`.  I think there's a few others like this throughout the page.

You might try regexp searching for word boundaries...?  If that doesn't work, cases like this might be difficult for automation to detect properly -- you might want to just highlight common words that may show up in paragraph text (e.g., "result") and print them out from your Python script and then manually check for that kind of thing afterwards.